### PR TITLE
server-1745 | Added commands and terraform steps

### PR DESCRIPTION
**Pull request by atulsingh0**
# Description
* Added Helm commands 
* Added step to run Nomad Terraform
* Removed cleanup step as it is now a part of exporter script 

# Reasons
[SERVER-1745](https://circleci.atlassian.net/browse/SERVER-1745)

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: https://circleci.com/docs/style/.

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [ ] Break up walls of text by adding paragraph breaks.
- [ ] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [ ] Keep the title between 20 and 70 characters.
- [ ] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [ ] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [ ] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

File: jekyll/_cci2/server/Installation/migrate-from-server-3-to-server-4.adoc [hunk 1]
```diff
@@ -171,22 +184,14 @@ The following Kubernetes service object is added:
...
-=== Step 8 - Validate your migration to server v4.x
-
-Re-run https://support.circleci.com/hc/en-us/articles/360011235534-Using-realitycheck-to-validate-your-CircleCI-installation[realitycheck] on your new server 4.x environment by pushing a fresh commit.
-
-[#step-9-execute-cleanup-step]
-=== Step 9 - Execute cleanup step
-Execute the following cleanup step to remove the `kots` annotation and pod:
+[#step-8-execute-nomad-terraform]
+=== Step 8 - Execute Nomad Terraform
+Execute the link:https://github.com/CircleCI-Public/server-terraform[Nomad Terraform] to re-create nomad client where `server_endpoint` value is set to be `<domain>:4647`.
```
Comment (rosieyohannan) [inline jekyll/_cci2/server/Installation/migrate-from-server-3-to-server-4.adoc:189]: @atulsingh0 Can we provide the commands to do this too?

Comment (atulsingh0) [reply jekyll/_cci2/server/Installation/migrate-from-server-3-to-server-4.adoc]: @rosieyohannan Added the Nomad Client Doc link.

File: jekyll/_cci2/server/Installation/migrate-from-server-3-to-server-4.adoc [hunk 2]
```diff
@@ -171,22 +184,14 @@ The following Kubernetes service object is added:
...
-=== Step 8 - Validate your migration to server v4.x
-
-Re-run https://support.circleci.com/hc/en-us/articles/360011235534-Using-realitycheck-to-validate-your-CircleCI-installation[realitycheck] on your new server 4.x environment by pushing a fresh commit.
-
-[#step-9-execute-cleanup-step]
-=== Step 9 - Execute cleanup step
-Execute the following cleanup step to remove the `kots` annotation and pod:
+[#step-8-execute-nomad-terraform]
+=== Step 8 - Execute Nomad Terraform
+Execute the link:https://github.com/CircleCI-Public/server-terraform[Nomad Terraform] to re-create nomad client where `server_endpoint` value is set to be `<domain>:4647`. You can follow the steps mentioned link:https://circleci.com/docs/2.0/server-4-install-build-services#nomad-clients[here].
```
Comment (circlery) [inline jekyll/_cci2/server/Installation/migrate-from-server-3-to-server-4.adoc:189]: I don't know what "execute the link" means. What are the specific things someone has to do here?

File: jekyll/_cci2/server/Installation/migrate-from-server-3-to-server-4.adoc [hunk 3]
```diff
@@ -153,11 +163,14 @@ kubectl -n <namespace> get pods
 
 [#step-7-update-dns-setting]
 === Step 7 - Update DNS setting
-Server 4.x migration is a destructive change for your DNS configuration. There is only one load-balancer/external-ip. You now need to update this in your DNS configuration.
+Server 4.x migration is a destructive change for your DNS configuration. There is only one `load-balancer/external-ip` service `circleci-proxy` which you need to update in the DNS configuration.
```
Comment (circlery) [inline jekyll/_cci2/server/Installation/migrate-from-server-3-to-server-4.adoc:166]:
```suggestion
Server 4.x migration is a destructive change for your DNS configuration. Server 4.x replaces the multiple DNS records with a single`load-balancer/external-ip` service, named `circleci-proxy` or `circleci-proxy-acm`. Retrieve the external IP and update your DNS records accordingly.
```

File: jekyll/_cci2/server/Installation/migrate-from-server-3-to-server-4.adoc (hunk 2 above)
Comment (atulsingh0) [reply jekyll/_cci2/server/Installation/migrate-from-server-3-to-server-4.adoc:189]: @circlery It is - Execute the "Nomad Terraform" , `link` is a markdown to add the hyperlink

**Comment by rosieyohannan** [comment]
@atulsingh0 sorry about this but once again I'm going to have to copy across your changes to a new branch. I'll tag you in the replacement PR when ready :-D

**Comment by rosieyohannan** [comment]
replacing with https://github.com/circleci/circleci-docs/pull/7028

**Pull request closed without merging**